tests/cli/helpers/utils.go

```diff
@@ -0,0 +1,22 @@
+package helpers
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/stretchr/testify/require"
+	"strings"
+	"testing"
+)
+
+func AddFlags(cmd string, flags []string) string {
```

Comment:
You have a `helpers.go` and a `utils.go` file. Although an anti-pattern, I'd recommend we stick to a single file for the "common" helper stuff.

Reply:
Sure, addressed this in latest commit